File: what2watch-api/app/schemas/recommendations.py
from pydantic import BaseModel

class Recommendation(BaseModel):
    title: str
    vote_count: int
    vote_average: float
    score: float


class RecommendationList(BaseModel):
    recommendations: list[Recommendation]


class ChatResponse(BaseModel):
    is_successful: bool
    error_message: str | None
    data: list[str] | None

File: what2watch-gui/src/schemas/recommendations.ts
export interface Recommendation {
  title: string;
  vote_count: number;
  vote_average: number;
  score: number;
}

export interface ChatResponse {
  is_successful: boolean;
  error_message?: string;
  data?: string[];
}

export interface ChatMessage {
  isUserMessage: boolean;
  message: string;
}
